pvlib/modelchain.py

```diff
@@ -311,7 +321,22 @@ def __init__(self, system, location,
         self.ac_model = ac_model
         self.aoi_model = aoi_model
         self.spectral_model = spectral_model
-        self.temp_model = temp_model
+
+        # TODO: deprecated kwarg temp_model. Remove in v0.8
+        if temp_model is not None:
+            warnings.warn('The temp_model keyword argument is deprecated. Use '
+                          'temperature_model instead', pvlibDeprecationWarning)
+            if temperature_model is None:
+                temperature_model = temp_model
+            elif temp_model == temperature_model:
+                raise warnings.warn('Provide only one of temperature_model or '
```

Comment:
I'm not sure what the intended behavior is here, but the line should either be `warnings.warn` or `raise ValueError`.

Reply:
intended to `warn`.